### PR TITLE
Timeline

### PR DESCRIPTION
**Pull request by toledoal**
This PR adds a basic timeline for the Curve Editor. 
Connects the currently selected keyframe in the editor. Allows to quickly move through the animation.

We still need to connect the animation to the timeline, so we can see the animation frame while playing.

timeline1

### CODE REVIEW

File: dist/preview release/what's new.md [hunk 1]
```diff
@@ -45,6 +45,7 @@
 - Curve Editor to view current entity's animations (Read only) ([pixelspace](https://github.com/devpixelspace))
 - Add support to update inspector when switching to new scene ([belfortk](https://github.com/belfortk))
 - Drag keyframes in Curve Editor ([pixelspace](https://github.com/devpixelspace))
+- Add timeline control to move through frames and keyframes ([pixelspace](https://github.com/devpixelspace))
```
Comment (deltakosh) [inline dist/preview release/what's new.md:48]: We should just mention one big line in the  major feature section: Animation curve editor

File: inspector/src/components/actionTabs/tabs/propertyGrids/animations/timeline.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        event.preventDefault();
+        this.props.onCurrentFrameChange(this.props.currentFrame + 1);
+        (this._scrollable.current as HTMLDivElement).scrollLeft = this.props.currentFrame * 5;
+
+    }
+
+    previousFrame(event: React.MouseEvent<HTMLDivElement>){
+        event.preventDefault();
+        if (this.props.currentFrame !== 0) {
+        this.props.onCurrentFrameChange(this.props.currentFrame - 1);
```
Comment (deltakosh) [inline inspector/src/components/actionTabs/tabs/propertyGrids/animations/timeline.tsx:39]: identation seems off

File: dist/preview release/what's new.md (hunk 1 above)
Comment (toledoal) [reply dist/preview release/what's new.md:48]: sure, it makes sense.

File: dist/preview release/what's new.md [hunk 3]
```diff
@@ -42,9 +42,8 @@
 - Added right click options to create PBR and Standard Materials ([Deltakosh](https://github.com/deltakosh))
 - Added support for recording GIF ([Deltakosh](https://github.com/deltakosh))
 - Popup Window available (To be used in Curve Editor) ([pixelspace](https://github.com/devpixelspace))
-- Curve Editor to view current entity's animations (Read only) ([pixelspace](https://github.com/devpixelspace))
+- Curve Editor to view selected entity's animations (Read only) ([pixelspace](https://github.com/devpixelspace))
```
Comment (deltakosh) [inline dist/preview release/what's new.md:45]: this should be in the Major feature list :) (upper in the file)

Comment (toledoal) [reply dist/preview release/what's new.md:45]: ah.. ok. 😃

**Pull request merged**